superset-frontend/packages/superset-ui-core/src/style/index.tsx

```diff
@@ -124,6 +124,11 @@ const defaultTheme = {
       light1: '#B3DEFE',
       light2: '#EFF8FE',
     },
+    shadow: {
```

Comment:
Will these be used thoughout the app as standard?

Reply:
@pkdotson nice catch, something I meant to run by design and missed! I removed changes to global style in in the create ant d table pr and changes will reflect here as well.